neuralprophet/time_net.py

```diff
@@ -347,30 +329,18 @@ def __init__(
                 self.covar_nets[covar] = covar_net
 
         # Regressors
+        # Migrating Regressors
         self.config_regressors = config_regressors
-        self.regressors_dims = utils.config_regressors_to_model_dims(config_regressors)
-        if self.regressors_dims is not None:
-            n_additive_regressor_params = 0
-            n_multiplicative_regressor_params = 0
-            for name, configs in self.regressors_dims.items():
-                if configs["mode"] not in ["additive", "multiplicative"]:
-                    log.error("Regressors mode {} not implemented. Defaulting to 'additive'.".format(configs["mode"]))
```

Comment:
Just a comment: We should make a consistent decision where to raise error messages if certain parameters are missing. But fine for now.